README.md

```diff
@@ -1,59 +1,292 @@
 # buildevents [![Build Status](https://travis-ci.org/honeycombio/buildevents.svg?branch=master)](https://travis-ci.org/honeycombio/buildevents)
 
-buildevents is a small binary used to help instrument Travis-CI builds. It is installed during the setup phase and then invoked as part of each step in order to visualize the build as a trace in Honeycomb
+buildevents is a small binary used to help instrument builds in a build system such as Travis-CI, CircleCI, Jenkins, and so on. It is installed during the setup phase and then invoked as part of each step in order to visualize the build as a trace in Honeycomb
 
-The trace that you get at the end has a span for every step of the travis build and a span for every instrumented command within each step. Here's an example showing a build that goes through the `install`, `before_script`, `script`, and `after_success` steps:
+The trace that you get at the end represents the entire build. It has spans for each section and subsection of the build, representing groups of actual commands that are run. The duration of each span is how long that stage or specific command took to run, and includes whether or not the command succeeded.
 
-![buildevents_trace](https://user-images.githubusercontent.com/361454/53279652-23e34700-36c7-11e9-876c-4dc716416393.png)
+Here's an example showing a build that ran on CircleCI. It goes through running go tests, setting up javascript dependencies, triggers js_build and poodle_test in parallel after dependencies are configured, and then continues off below the captured portion of the waterfall.
+
+![CircleCI_Build_Trace](https://user-images.githubusercontent.com/361454/57872910-ac9eea00-77c1-11e9-8bdd-db7a870dcd61.png)
 
 # Setup
 
-You need to add an environment variable to your Travis build (done through the Travis UI) to set the API key to use to send the trace to Honeycomb. Set `BUILDEVENT_APIKEY` to your Honeycomb API key (available at https://ui.honeycomb.io/account)
+Getting your build ready to use `buildevents` involves:
+* installing the `buildevents` binary in your build environment
+* setting a number of environment variables for configuring the tool
+* choosing a unique trace identifier
+
+## Installation
+
+If you have a working go environment in your build, the easiest way to install `buildevents` is via `go install`.
+
+```
+go get https://github.com/honeycombio/buildevents/
```

Comment:
go get => go install?

Reply:
Go get does install too! Can you do an install without a get? Will look.